### ISSUE TITLE
A small error: function define() "opitimizer" should be "optimizer"

### ISSUE THREAD

**Issue by EvannaB** [huyng/incertae#2]
model.compile(opitimizer=optim, loss=gaussian_nll)
=>
model.compile(optimizer=optim, loss=gaussian_nll)
